bucket/windowtextextractor.json

```diff
@@ -0,0 +1,19 @@
+{
+    "version": "1.15.0",
+    "description": "A tool allows you to get a text from any window of an operating system including asterisk passwords.",
+    "homepage": "https://github.com/AlexanderPro/WindowTextExtractor",
+    "license": "MIT",
+    "url": "https://github.com/AlexanderPro/WindowTextExtractor/releases/download/v1.15.0/WindowTextExtractor_v1.15.0.zip",
+    "hash": "454f477dd62a2092a2aa09e751bc6a70cdb847a25d060b7e57bf7b8a426ba3c6",
+    "shortcuts": [
```

Comment:
```suggestion
    "hash": "454f477dd62a2092a2aa09e751bc6a70cdb847a25d060b7e57bf7b8a426ba3c6",
    "pre_install": [
        "if ($architecture -eq '64bit') {",
        "    Rename-Item \"$dir\\WindowTextExtractor.exe\" 'WindowTextExtractor32.exe'",
        "    Rename-Item \"$dir\\WindowTextExtractor64.exe\" 'WindowTextExtractor.exe'",
        "}"
    ],
    "shortcuts": [
```

Reply:
WindowTextExtractor.exe is for 32 bit and 64 bit OS. WindowTextExtractor64.exe is a special file and not for a manual run.

Reply:
sorry, I did not run it for check.